apps/api-journeys/src/app/modules/integration/growthSpaces/growthSpaces.service.ts

```
@@ -0,0 +1,197 @@
+    if (languageCode == null) {
+      const { data } = await apollo.query({
+        query: gql`         
+            query Language($languageId: ID!) {
+            language(id: $languageId) {
+              bcp47
+              id
+            }
+          }
+        `,
+        variables: { languageId: journey?.languageId }
+      })
+      if (data?.language?.bcp47 == null)
+        throw new GraphQLError('cannot find language code', {
+          extensions: { code: 'NOT_FOUND' }
+        })
+      languageCode = data?.language?.bcp47
+      await this.cacheManager.set(key, languageCode, ONE_DAY_MS)
+    }
+
+    if (
+      integration?.accessId == null ||
+      integration?.accessSecretCipherText == null ||
+      integration?.accessSecretIv == null ||
+      integration?.accessSecretTag == null
+    )
+      throw new GraphQLError(
+        'incorrect access Id and access secret for Growth Space integration',
+        {
+          extensions: { code: 'UNAUTHORIZED' }
+        }
+      )
+
+    const decryptedAccessSecret =
+      await this.integrationService.decryptSymmetric(
+        integration.accessSecretCipherText,
+        integration.accessSecretIv,
+        integration.accessSecretTag,
+        process.env.INTEGRATION_ACCESS_KEY_ENCRYPTION_SECRET
+      )
+
+    const body = JSON.stringify({
+      subscriber: {
+        route_id: block.routeId,
+        language_code: languageCode,
+        email,
+        first_name: name
+      }
+    })
+
+    try {
+      await fetch('https://api.growthspaces.org/api/v1/subscribers', {
+        method: 'POST',
+        headers: {
+          'Access-Id': integration.accessId,
+          'Access-Secret': decryptedAccessSecret,
+          'Content-Type': 'application/json'
+        },
+        body
+      })
+    } catch (e) {
+      throw new GraphQLError(
+        e.message ?? 'failed to fetch from Growth Spaces',
+        {
+          extensions: { code: 'INTERNAL SERVER ERROR' }
+        }
+      )
```

again we don't care if this fails. fail silently. maybe add a console.error log for datadog.